src/java.desktop/share/classes/java/awt/image/ColorConvertOp.java

```diff
@@ -77,13 +79,6 @@
     boolean          gotProfiles;
     float[]          srcMinVals, srcMaxVals, dstMinVals, dstMaxVals;
 
-    /* the class initializer */
-    static {
-        if (ProfileDeferralMgr.deferring) {
-            ProfileDeferralMgr.activateProfiles();
```

Comment:
Seems unnecessary, this class uses the `sun.java2d.cmm.ColorTransform` which will load the profile when necessary.